src/node_data.js

```diff
@@ -127,10 +127,15 @@ const getData = function(node) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const importNode = function(node) {
+  let data = node['__incrementalDOMData'];
```

Comment:
Hm? Do you mean https://github.com/jridgewell/incremental-dom/blob/6b5aadbea2bf10687230c055d6fc1abaf1b2060a/src/core.js#L94?